src/main/java/com/bookmanagement/BookManagement/serviceimplementation/BookServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.bookmanagement.BookManagement.serviceimplementation;
+
+import com.bookmanagement.BookManagement.dto.BookDto;
+import com.bookmanagement.BookManagement.entity.Book;
+import com.bookmanagement.BookManagement.mapper.BookMapper;
+import com.bookmanagement.BookManagement.repository.BookRepository;
+import com.bookmanagement.BookManagement.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service("bookService")
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
+
+    @Autowired
+    public BookServiceImpl(BookRepository bookRepository, BookMapper bookMapper) {
+        this.bookRepository = bookRepository;
+        this.bookMapper = bookMapper;
+    }
+
+    @Override
+    public List<BookDto> getAllBooks() {
+        List<Book> temp = bookRepository.findAll();
```

Comment:
```suggestion
        List<Book> bookEntities = bookRepository.findAll();
```
Please use intuitive and relatable names.

Reply:
Done